src/app/app.component.scss

```diff
@@ -0,0 +1,6 @@
+.weather-container{
+    display: flex;
+    flex-direction: row;

```

Comment:
`flex-direction: row;" is the default. No need to specify it. 
Remove this line.